mailbox/api/src/test/java/org/apache/james/mailbox/MailboxManagerTest.java

```diff
@@ -61,6 +61,9 @@
  */
 public abstract class MailboxManagerTest {
 
+    protected static final int LIMIT_ANNOTATIONS = 3;
+    protected static final int LIMIT_ANNOTATION_SIZE = 30;
```

Comment:
never used?

Reply:
Will be used by memory and cassandra who for now duplicates them